src/dfcx_scrapi/tools_ml/utterance_generator_utils.py

```diff
@@ -195,46 +238,59 @@ def _generate_phrases(self, training_phrases: pd.DataFrame, set_size: int):
     def create_synthetic_set(
         self, agent_id: str, intent_subset: list, set_size: int = 100
     ):
-        """Create a test set where none of the utterances in the test set
-        are in the training of the existing phrases"""
+        """Creates a synthetic test set.
+        
+        Creates a test set where none of the utterances in the test set
+        are in the training of the existing phrases.
+
+        Args: 
+            agent_id: (str) ID of the DFCX agent.
+            intent_subset: (list) intents to generate a test set for.
+            set_size: (int) number of synthetic phrases to generate. 
+                Default 100. 
+        Returns:
+            a DataFrame containing synthetic test set utterances.
+        """
         training_phrases = self.intents.bulk_intent_to_df(
             agent_id=agent_id, intent_subset=intent_subset
         )
-        training_phrases = training_phrases.copy().rename(columns={"tp": "utterance"})
+        training_phrases = training_phrases.copy().rename(
+            columns={"tp": "utterance"})
 
         test_set = self._generate_phrases(training_phrases, set_size)
         test_set = test_set[:set_size]
         return test_set.reset_index(drop=True)
 
-    def create_test_set(self, agent_id: str, intent_subset: list, set_size: int = 100):
-        """
-        Create a test set for a given list of intents. The phrases in this set will not
-        be exact string match phrases which exist in the training phrases but will be close
-        semantically.This set is automatically labeled by the intent whose training was
-        used to generate the new phrase. This can be used to run through the core.conversations
-        run_intent_detection function. You may need to specify a flow_display_name and
-        page_display_name in the dataframe to run the set at the correct location.
-
+    def create_test_set(
```

Comment:
@Hgithubacct same here.

Perhaps `create_test_dataset`
Also update the `set_size` arg to `dataset_size`